packages/google-cloud-bundle/README.md

```diff
@@ -0,0 +1,34 @@
+# Shopsys Google Cloud Bundle
+
+[![Build Status](https://travis-ci.org/shopsys/google-cloud-bundle.svg?branch=master)](https://travis-ci.org/shopsys/google-cloud-bundle)
+[![Downloads](https://img.shields.io/packagist/dt/shopsys/google-cloud-bundle.svg)](https://packagist.org/packages/shopsys/google-cloud-bundle)
+
+This bundle is used to allow Shopsys Framework](https://www.shopsys-framework.com) integration with Google Cloud.
+
+This repository is maintained by [shopsys/shopsys] monorepo, information about changes are in [monorepo CHANGELOG.md](https://github.com/shopsys/shopsys/blob/master/CHANGELOG.md).
+
+## Installation
+The plugin is a Symfony bundle and is installed in the same way:
+
+### Download
+First, you download the package using [Composer](https://getcomposer.org/):
```

Comment:
It sound like a second section, perhaps *Bundle registration*, is missing. It should cover adding the bundle into the list in `AppKernel.php`.